### ISSUE TITLE
Question about learning rate

### ISSUE THREAD

**Issue by David-Zhao-1997** [princeton-vl/RAFT-Stereo#35]
We prepare to cite your paper in our new work. When I reproduce your work, I noticed that the **maximum** learning rate of training Sceneflow is 0.0002 in your provided code. In your paper, It is said that Sceneflow is trained with a **minimum** learning rate of 1e-4. Should I just keep the parameter 0.0002 in the code to reproduce your work?

Thanks!

**Comment by lahavlipson** [comment]
You can keep the default learning rate in our code to reproduce our results.

**Comment by David-Zhao-1997** [comment]
Thanks for sharing such a perfect work!

**Comment by David-Zhao-1997** [comment]
When I reproduce your work. I find it difficult to train middlebury. Could you please help me figure it out.
1. What's the learning rate of fine-tune?
2. 
> we fine-tune on 384x1000 random crops
of the 23 Middlebury traning images for 4000 steps
When I try to use 23 images downloaded from middlebury 2014, the dataloader report no mask0nocc. But it is fine with middv3  15 training images provided by MiddEval3-data-F.zip. It is OK to reproduce your work by only applying 15 training pairs?

Thanks!

**Comment by lahavlipson** [comment]
The invalid pixels should be stored as infs in the .pfm files. You may need to modify readDispMiddlebury for this special case where mask0nocc isn't available, sorry for the inconvenience. I'll push a fix for this.

**Comment by David-Zhao-1997** [comment]
Could you please tell me which learning rate do you use for fine-tune？

**Comment by lahavlipson** [comment]
2e-5

**Comment by lahavlipson** [comment]
FYI @David-Zhao-1997 I have just added a script to download all of the finetuning data and updated the README to specify exactly which command to run.
